compiler/rustc_pattern_analysis/src/usefulness.rs

```diff
@@ -711,8 +711,8 @@
 use std::fmt;
 
 #[cfg(feature = "rustc")]
+use rustc_data_structures::fx::{FxHashMap, FxHashSet};
 use rustc_data_structures::stack::ensure_sufficient_stack;
```

Comment:
This too breaks the way compilation on stable is handled.

Reply:
I see. So the best we can do for now is to change this on nightly?

Reply:
Using `rustc_hash::FxBuildHasher` would work on stable. Centralizing all `rustc_hash` imports to `rustc_data_structures::fx` can only be done on nightly while rustc_data_structures needs nightly.

Reply:
Or we could create a `rustc_hash_facade` crate that would unify all the imports.